.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 >^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 > Remember to delete this note before submitting your pull request.
 >
-> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
+> For pull requests on 1.16 Features: set Milestone to 1.16 and Base Branch to dev-1.16
```

Comment:
Should this say “**1.17**”?

1.16 is released so documentation for that lives in `master`.

Reply:
1.17 seems more appropriate here but the handbook use of ```future release``` seems to imply 1.16(based on previous usage). @jimangel  thoughts?

Reply:
Good catch, should be 1.17 (and dev-1.17). I think the hand book can be clearer... It also gets funky post-release, because now all of the "future / current" release concepts are +1 (or are they? depending on how you read it :stuck_out_tongue: ) ...